src/frontend/identity/index.ts

```diff
@@ -31,6 +60,17 @@ export class Identity extends KojiBridge {
     return hasGrants;
   }
 
+  /**
+   * Requests the specified authorization grants from the user for the Koji.
+   *
+   * @param   grants           List of {@link AuthGrantCapability| authorization grants} to request from the user.
+   * @param   usageDescription Custom message to display when requesting the grant.
+   * @return                   {@link UserToken}
```

Comment:
Similar here, I think we should reserve `@link` for use in descriptions, as these should be autolinked in the generated documentation if we do something like `@return {UserToken}`

Reply:
That _is_ a description. We don't need to explicitly include the types in the doc comments because they are defined by the typescript already. For the UserToken in particular, I have an open question about how to reuse the description. I tried including some text in the global definition, but it didn't seem to get picked up by the HTML parser. It is in the JSON so maybe that's something we can use or test differently.